packages/react/src/components/form/Checkbox/Group/Group.tsx

```diff
@@ -54,7 +54,9 @@ export const CheckboxGroup = forwardRef<
         ? currentValue.filter((x) => x !== checkboxValue)
         : [...currentValue, checkboxValue];
 
-      value ?? setInternalValue(updatedValue);
+      if (typeof value !== 'undefined' || value !== null) {
```

Comment:
Jeg tror dette er akkurat det samme som å skrive `value != null` eller `value != undefined`. Det skal bli `true` hvis `value` er noe annet enn `null` eller `undefined` (også for `""`, `0` og `[]`). Men det er jo fint å være på den sikre siden, JS-operatorer er jo ikke alltid så intuitive 😅

Reply:
hehe, ja, som du sier, føler det kan fort går galt om enn skal belage seg på at alle etter enn kan alle finurlighetene til javascript 🙈

Reply:
Kunne laget en `isNil` funkjson i utils 🤔